src/codegen/replacements.ts

```diff
@@ -81,95 +128,103 @@ export const enums = {
 export const warnOnIdentifiersNotPresentAtRuntime = [
   //
   "OutOfMemoryError",
-  "notImplementedIssue",
   "notImplementedIssueFn",
 ];
 
 // These are passed to --define to the bundler
 const debug = process.argv[2] === "--debug=ON";
 export const define: Record<string, string> = {
-  "process.env.NODE_ENV": JSON.stringify(debug ? "development" : "production"),
-  "IS_BUN_DEVELOPMENT": String(debug),
-
+  IS_BUN_DEVELOPMENT: String(debug),
   $streamClosed: "1",
   $streamClosing: "2",
   $streamErrored: "3",
   $streamReadable: "4",
   $streamWaiting: "5",
   $streamWritable: "6",
-
-  "process.platform": JSON.stringify(Bun.env.TARGET_PLATFORM ?? process.platform),
   "process.arch": JSON.stringify(Bun.env.TARGET_ARCH ?? process.arch),
+  "process.env.NODE_ENV": JSON.stringify(debug ? "development" : "production"),
+  "process.platform": JSON.stringify(Bun.env.TARGET_PLATFORM ?? process.platform),
 };
 
 // ------------------------------ //
 
 for (const name in enums) {
   const value = enums[name];
-  if (typeof value !== "object") throw new Error("Invalid enum object " + name + " defined in " + import.meta.file);
-  if (typeof value === null) throw new Error("Invalid enum object " + name + " defined in " + import.meta.file);
-  const keys = Array.isArray(value) ? value : Object.keys(value).filter(k => !k.match(/^[0-9]+$/));
```

Comment:
`k.match` produces an array or null. In this case we just needed a boolean to `regexp.test` is a better fit.